java/src/main/java/com/datasqrl/ai/models/openai/ChatModel.java

```diff
@@ -1,4 +1,4 @@
-package com.datasqrl.ai;
+package com.datasqrl.ai.models.openai;
```

Comment:
One thing we have to figure out is how we want to generalize this to other models. Right now, the model specific stuff is in the models sub-packages. And the SimpleServer and CmdLineChatBot are OpenAI specific.

Since Groq and others are supporting the OpenAI API spec, it probably makes sense to generalize this to those models.
We also need a GenericTestChatBot that we can use to execute the same test suites against all the models.

How would you structure this?

Reply:
I think the details will become apparent after I add one or two more models. Let's leave it like this for now.